src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Be careful, the excavator started digging a hole!!!");
```

Comment:
The message in the 'doWork' method should indicate what type of Machine is working. In this case, it should clearly state that it is the 'Excavator'. Consider rephrasing to something like 'Excavator is working - digging a hole!' to make it more informative.